Libraries/src/Amazon.Lambda.AspNetCoreServer/APIGatewayProxyFunction.cs

```diff
@@ -244,6 +245,18 @@ protected override void MarshallRequest(InvokeFeatures features, APIGatewayProxy
                 // was marshalled into ASP.NET Core request.
                 PostMarshallConnectionFeature(connectionFeatures, apiGatewayRequest, lambdaContext);
             }
+
+            {
+                var tlsConnectionFeature = (ITlsConnectionFeature) features;
+                var clientCertPem = apiGatewayRequest?.RequestContext?.Identity?.ClientCert?.ClientCertPem;
+                if (clientCertPem != null)
+                {
+                    // Remove "--------BEGIN CERTIFICATE-----\n" and "-----END CERTIFICATE-----"
+                    clientCertPem = clientCertPem.Substring(28, clientCertPem.Length - 53);
```

Comment:
Same comment as above about verify the cert. Probably put it a utility method that converts incoming cert into the X509Certificate2.

Reply:
See comment above https://github.com/aws/aws-lambda-dotnet/pull/787#discussion_r553662430